PalCalc.Solver/BreedingSolver.cs

```diff
@@ -3,13 +3,21 @@
 using PalCalc.Solver.ResultPruning;
 using Serilog;
 using System;
+using System.Collections.Concurrent;
+using System.Collections.Frozen;
 using System.Collections.Generic;
 using System.Diagnostics;
 using System.Linq;
 using System.Numerics;
 using System.Text;
 using System.Threading.Tasks;
 
+/*
+ * TODO
+ * 
+ * - Add warning when memory size likely exceeds system limits
+ */
```

Comment:
Pending TODO

Since we've got `MemoryMonitor` implemented, this could just be a warning in UI if too many options are selected

Reply:
Ignoring for now